pkg/cmd/admin/policy/create.go

```diff
@@ -0,0 +1,104 @@
+package policy
+
+import (
+	"errors"
+	"fmt"
+	"io"
+
+	"github.com/spf13/cobra"
+
+	kcmdutil "k8s.io/kubernetes/pkg/kubectl/cmd/util"
+
+	authapi "github.com/openshift/origin/pkg/authorization/api"
+	"github.com/openshift/origin/pkg/client"
+	"github.com/openshift/origin/pkg/cmd/templates"
+	"github.com/openshift/origin/pkg/cmd/util/clientcmd"
+)
+
+const CreateClusterRoleRecommendedName = "create-cluster-role"
+
+var createClusterRoleExample = templates.Examples(`
+	# TODO
+	%[1]s dev --resources a,b,c --actions x,y,z`)
+
+type NewClusterRoleOptions struct {
+	ClusterRoleClient client.ClusterRoleInterface
+
+	Name      string
+	Resources []string
+	Actions   []string
+}
+
+// NewCmdCreateClusterRole implements the OpenShift cli create-cluster-role command.
+func NewCmdCreateClusterRole(name, fullName string, f *clientcmd.Factory, out io.Writer) *cobra.Command {
+	options := &NewClusterRoleOptions{}
+
+	cmd := &cobra.Command{
+		Use:     name + " <role-name>", // TODO
+		Short:   "Create a new cluster role",
+		Long:    `Create a new cluster role for specified resources and actions`, // TODO
+		Example: fmt.Sprintf(createClusterRoleExample, fullName),
+		Run: func(cmd *cobra.Command, args []string) {
+			if err := options.Complete(f, args); err != nil {
+				kcmdutil.CheckErr(kcmdutil.UsageError(cmd, err.Error()))
+			}
+			kcmdutil.CheckErr(options.Validate())
+			kcmdutil.CheckErr(options.CreateRole())
+		},
+	}
+
+	cmd.Flags().StringSliceVarP(&options.Resources, "resources", "", options.Resources, "list of resources (separated by comma)")
+	cmd.Flags().StringSliceVarP(&options.Actions, "actions", "", options.Actions, "list of actions (separated by comma)")
+
+	return cmd
+}
+
+func (o *NewClusterRoleOptions) Complete(f *clientcmd.Factory, args []string) error {
+	if len(args) == 0 {
+		return errors.New("you must specify role name")
+	}
+
+	o.Name = args[0]
+
+	osClient, _, err := f.Clients()
+	if err != nil {
+		return err
+	}
+
+	o.ClusterRoleClient = osClient.ClusterRoles()
+
+	return nil
+}
+
+func (o *NewClusterRoleOptions) Validate() error {
+	if len(o.Name) == 0 {
+		return errors.New("Name is required")
+	}
+	if len(o.Resources) == 0 {
+		return errors.New("Resources is required")
+	}
+	if len(o.Actions) == 0 {
+		return errors.New("Actions is required")
+	}
+	return nil
+}
+
+func (o *NewClusterRoleOptions) CreateRole() error {
+	rule, err := authapi.NewRule(o.Actions...).Resources(o.Resources...).Groups("").Rule()
```

Comment:
we'll likely need to express distinct verb/resource pairings ("create a role that lets me get,list,watch pods and create,update,delete,list,watch replicasets")